targets/TARGET_STM/TARGET_STM32F4/device/stm32f4xx_hal_dma.c

```diff
@@ -761,6 +761,9 @@ void HAL_DMA_IRQHandler(DMA_HandleTypeDef *hdma)
       
       /* Update error code */
       hdma->ErrorCode |= HAL_DMA_ERROR_TE;
+
+      /* Change the DMA state */
+      hdma->State = HAL_DMA_STATE_ERROR; // FIX
```

Comment:
One nit pick: is the `// FIX` comment a convention you use? It seems a bit nondescript.

I would either leave the comment out or add more description to it. (same goes for the other `// FIX` comments)